### PR TITLE
Make a Form for admins to accept Bids #2093

### PR DESCRIPTION
**Pull request by Sarthak5598**
#2093

### CODE REVIEW

File: website/templates/bid_selection.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                        bidContainer.appendChild(bidElement);
+                    });
+                }
+            })
+            .catch(error => console.error('Error fetching bids:', error));
+        }
+
+        function changeStatus(bidId) {
+            const csrftoken = getCookie('csrftoken');
+            fetch(`/change-bid-status/`, {
```
Comment (arkid15r) [inline website/templates/bid_selection.html:107]: Don't forget to change it here too

File: website/views.py [hunk 2]
```diff
@@ -4417,6 +4417,42 @@ def generate_bid_image(request, bid_amount):
...
+
+            all_bids = Bid.objects.filter(issue_url=issue_url).values()
+            return JsonResponse(list(all_bids), safe=False)
+        except json.JSONDecodeError:
+            return JsonResponse({"success": False, "error": "Invalid JSON"})
+    return HttpResponse(status=405)
+
+
+@method_decorator(user_passes_test(lambda u: u.is_superuser), name="dispatch")
+def selectBid(request):
```
Comment (arkid15r) [inline website/views.py:4452]:
```suggestion
def select_bid(request):
```

**Review by arkid15r — CHANGES_REQUESTED** [review]
Hi @Sarthak5598, 

could you look into the comments below when you get a chance:

**Review by DonnieBLT — CHANGES_REQUESTED** [review]

File: website/views.py [hunk 3]
```diff
@@ -4417,6 +4417,41 @@ def generate_bid_image(request, bid_amount):
...
+            bid = Bid.objects.get(id=bid_id)
+            bid.status = "Selected"
+            bid.save()
+            return JsonResponse({"success": True})
+        except Bid.DoesNotExist:
+            return JsonResponse({"success": False, "error": "Bid not found"})
+    return HttpResponse(status=405)
+
+
+@csrf_exempt
```
[the remark this answers is not in the document]
Comment (Sarthak5598) [reply website/views.py]: **Done!**

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**